gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -500,6 +512,16 @@ def test_pipeline_read_single_file_large(self):
     assert_that(pcoll, asserts.count_equals_to(5))
     pipeline.run()
 
+  @unittest.skipIf(VCF_FILE_DIR_MISSING, 'VCF test file directory is missing')
+  def test_pipeline_read_all_single_file_large(self):
+    pipeline = TestPipeline()
+    pcoll = (pipeline
+             | 'Create' >> beam.Create(
+                 [testdata_util.get_full_file_path('valid-4.0.vcf')])
+             | 'Read' >> ReadAllFromVcf())
+    assert_that(pcoll, asserts.count_equals_to(5))
+    pipeline.run()
```

Comment:
For the large test it may be better to run on valid-4.1-large.vcf since the 4.0 file isn't much different in size to the sample lines.

Reply:
Done.